crates/wasi-http/src/types_impl.rs

```diff
@@ -886,7 +885,7 @@ impl<T: WasiHttpView> crate::bindings::http::types::HostRequestOptions for T {
         Ok(id)
     }
 
-    fn connect_timeout_ms(
+    fn connect_timeout(
```

Comment:
I think the internals of all these methods also need to change since right now they're using `*_millis`-related `Duration` methods

Reply:
Good catch, thanks!